fastparquet/test/test_output.py

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
I don't really like this. Python 2-compatible APis should accept Python 2 `str` in most places (for example column names).